projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -156,8 +157,18 @@ export class TuiTiptapEditorService extends TuiEditor {
         this.editor.chain().focus().toggleCodeBlock().run();
     }
 
-    isActive(name: string): boolean {
-        return this.editor.isActive(name);
+    isActive(nameOrAttributes: Record<string, string> | string): boolean {
+        return this.editor.isActive(nameOrAttributes);
+    }
+
+    isActive$(
+        nameOrAttributes: Record<string, string> | string,
+    ): Observable<boolean | null> {
+        return this.stateChange$.pipe(
+            map(() => this.isActive(nameOrAttributes)),
+            distinctUntilChanged(),
+            startWith(null),
```

Comment:
Wrong order

Reply:
fixed

Reply:
oh, i forgot to change signature of function

Reply:
fixed again :)